docs/waivers.md

```diff
@@ -38,7 +36,7 @@ control_id:
 + `justification` can be any text you want and might include a reason
   as well as who signed off on the waiver.
 
-### Example:
+### Examples:
```

Comment:
I went singular because this is one file. (I don't care either way, just explaining)